content/javascript/concepts/dates/terms/toLocaleTimeString/toLocaleTimeString.md

```diff
@@ -0,0 +1,102 @@
+---
+Title: '.toLocaleTimeString()'
+Description: 'Returns a modified string from a given Date object, usually for events. It is translated to a specific language format according to an event locale and other options.'
+Subjects:
+  - 'Web Development'
+  - 'Computer Science'
+Tags:
+  - 'Date'
+  - 'Methods'
+CatalogContent:
+  - 'introduction-to-javascript'
+  - 'paths/front-end-engineer-career-path'
+---
+
+The `.toLocaleTimeString()` date method returns a modified string of a given [`Date` object](https://www.codecademy.com/resources/docs/javascript/dates), usually for events. It is translated to a specific language according to an event's `locale` and other `options`.
+
+## Syntax
+
+```js
+const myEventDate = new Date('December 31, 2021 15:00:00');
+
+const locale = 'en-US';
+
+const options = {
+	hour: 'numeric',
+              minute: 'numeric'
+    	second: 'numeric',
+};
+
+myEventDate.toLocaleTimeString(locale, options);
+```
+
+## Example Without Parameters
+
+When used without any parameters, `.toLocaleTimeString()` returns a string with the `hour`, `minute`, and `second` options defaulted to `numeric`. How they're arranged and formatted depends on the default `locale` in which the method was used.
+
+In the example below, `.toLocaleTimeString()` is being used in the `en-US` locale (more specifically, in the `America/Los_Angeles` timezone).
+
+```js
+const date = new Date(2021, 12, 31 2021 15:00:00);
+
+console.log(date.toLocaleTimeString());
+// Output: 15:00:00
```

Comment:
```suggestion
const date = new Date(2021, 11, 31, 15, 0, 0);

console.log(date.toLocaleTimeString());
```
I changed the code a little bit because the previous code was giving an error. This code should result in this output - `3:00:00 PM`. Add it below the codeblock in the ` ```shell` block.